setup.py

```diff
@@ -30,9 +30,9 @@
     "pycodestyle<2.7.0,>=2.6.0",
     "pep8-naming>=0.3.3,<=0.10.0",
     "pydocstyle>=2.0.0",
-    "pylint==2.5.3",
+    "pylint==2.7.2",
```

Comment:
```suggestion
    "pylint==2.8.3",
```
astroid is pinned to 2.5.6 in pylint 2.8.3 and will stay pinned in future versionso I don't know what to do with the astroid version line 35, but the easiest would be to use pylint's version.

Reply:
One additional comment - is there any specific reason that prospector is pinning 2.8.3/2.7.2? It seems like prospector is pretty widely compatible, so it might increase user flexibility to unpin asteroid, and change the pylint version to something like:

```
pylint>=2.5.3,<2.9
```

This would increase user flexibility, allowing users requiring a fixed pylint/asteroid version to specify exactly (unless there are actually breaking changes).